docker/local/django/Dockerfile

```diff
@@ -1,60 +1,59 @@
-ARG BASE_IMAGE=python:3.9-alpine3.16
+ARG BASE_IMAGE=python:3.10-slim
```

Comment:
As discussed - lets explore build stages to build debian images for local and alpine for other envs